_includes/fonts.css

```diff
@@ -1,10 +1,50 @@
-
-/* inter-variable */
 @font-face {
-	font-family: Inter Variable;
-	font-style: normal;
-	font-weight: 100 1000;
```

Comment:
Did we have an entire Base64 encoded font file embedded in the CSS?  :O

Reply:
Let's now dwell on what once was ;)

Reply:
Don't get me wrong.  I was impressed and couldn't believe my eyes!  :)

Reply:
Let's just say `gtksourceview` did not share your enthusiasm ;)